### PR TITLE
[new release] reparse, reparse-lwt and reparse-lwt-unix (3.0.0)

### PR DESCRIPTION
**Pull request by bikallem**
Recursive descent parsing library for ocaml

- Project page: <a href="https://github.com/lemaetech/reparse">https://github.com/lemaetech/reparse</a>

##### CHANGES:

- Overhaul parser implementation - use functor based implementation. Introduce, `Make_buffered_input`, `Make_unbuffered_input` and `Make` functors.
- Remove `reparse-unix` package
- Remove base dependency
- Facilitate IO promise monads such as `Lwt` and `Async`
- Add package `reparse-lwt` which defines `Lwt_stream.t` as one of the input sources.
- Add package 'reparse-lwt-unix' which defines `Lwt_unix.file_descr` and `Lwt_io.input_channel` as parser input sources.
-

### CODE REVIEW

**Comment by bikallem** [comment]
@kit-ty-kate how do I make the ci run again? I tried closing and opening the PR but it doesn't seem to run.

**Comment by kit-ty-kate** [comment]
I clicked the rebuild button an hour ago but it's far in the queue. For next time you can trigger a rebuild yourself by clicking on the rebuild button on the job page you want to rebuild.

**Comment by bikallem** [comment]
Thanks. What is the url for the jobs page?

**Comment by kit-ty-kate** [comment]
Just click on `Details` then the failing job you want to rebuild
scrn-2021-06-24-14-19-50
scrn-2021-06-24-14-20-02
scrn-2021-06-24-14-20-19

File: packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Reparse lwt-unix based input support"
+description: "Reparse lwt-unix based input support"
```
Comment (kit-ty-kate) [inline packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam:3]: The `synopsis` takes precedence over `description`. For next time could you elaborate a little more in the description?

Comment (kit-ty-kate) [reply packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam:3]: or remove description, this would also be totally fine of course

**Comment by kit-ty-kate** [comment]
Thanks!

File: packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam (hunk 1 above)
Comment (bikallem) [reply packages/reparse-lwt-unix/reparse-lwt-unix.3.0.0/opam:3]: sure.

**Pull request merged**